public/app/features/plugins/admin/components/ConnectionsTab.tsx

```diff
@@ -0,0 +1,106 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { EmptyState, Stack, TextLink, useStyles2 } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+import { contextSrv } from 'app/core/services/context_srv';
+import { ViewProps } from 'app/features/datasources/components/DataSourcesList';
+import { DataSourcesListCard } from 'app/features/datasources/components/DataSourcesListCard';
+import { getDataSources, useLoadDataSources } from 'app/features/datasources/state';
+import { AccessControlAction, useSelector } from 'app/types';
+
+import { CatalogPlugin } from '../types';
+
+import { GetStartedWithDataSource } from './GetStartedWithPlugin/GetStartedWithDataSource';
+
+interface Props {
+  plugin: CatalogPlugin;
+}
+
+export default function ConnectionsTab({ plugin }: Props) {
+  const { isLoading } = useLoadDataSources();
+
+  const allDataSources = useSelector((state) => getDataSources(state.dataSources));
+  const dataSources = allDataSources.filter((ds) => ds.type === plugin.id);
+  const hasWriteRights = contextSrv.hasPermission(AccessControlAction.DataSourcesWrite);
+  const hasExploreRights = contextSrv.hasAccessToExplore();
+
+  return (
+    <ConnectionsList
+      dataSources={dataSources}
+      dataSourcesCount={dataSources.length}
+      isLoading={isLoading}
+      plugin={plugin}
+      hasWriteRights={hasWriteRights}
+      hasExploreRights={hasExploreRights}
+    />
+  );
+}
+
+type ListProps = Omit<ViewProps, 'hasCreateRights'> & {
+  plugin: CatalogPlugin;
+};
+
+export function ConnectionsList({
+  dataSources,
+  dataSourcesCount,
+  isLoading,
+  hasWriteRights,
+  hasExploreRights,
+  plugin,
+}: ListProps) {
+  const styles = useStyles2(getStyles);
+
+  if (!isLoading && dataSourcesCount === 0) {
+    return (
+      <EmptyState
+        variant="call-to-action"
+        button={<GetStartedWithDataSource plugin={plugin} />}
+        message={t('data-source-list.empty-state.title', 'No data sources defined')}
+      />
+    );
+  }
+
+  const getDataSourcesList = () => {
+    if (isLoading) {
+      return new Array(20)
```

Comment:
nit: is there a specific reason for this skeleton to have 20 items? do we also do the same in the other connections page? 20 might make the page show a vertical scroll for a moment just to maybe load 2 or 3 or nothing.

Reply:
`ConnectionsList` component is pretty much same as https://github.com/grafana/grafana/blob/main/public/app/features/datasources/components/DataSourcesList.tsx but without header and `EmptyState` component. And the content height is similar and 20 was used there as well, so.